include/deal.II/lac/trilinos_sparsity_pattern.h

```diff
@@ -1230,6 +1230,11 @@ namespace TrilinosWrappers
      */
     std::unique_ptr<Epetra_CrsGraph> nonlocal_graph;
 
+    /**
+     * The set of locally owned rows.
+     */
+    IndexSet locally_owned_rows;
```

Comment:
This duplicates information that the Trilinos object already stores, and furthermore the way you fill it only supports the case where the Trilinos object only stores a single, contiguous range. (The latter may be ok -- I don't think we ever use anything else.) But I would prefer if we had a way to query the Trilinos object whether a row is locally stored, rather than duplicate this information (and risk things being out of sync).

Reply:
Actually, I save the correct `IndexSet` if it is provided in the constructor or a `reinit` function.
The other interfaces (taking are `Epetra_Map`) are deprecated and I assume to have one contiguous range indeed.